extensions/amp-story-interactive/0.1/amp-story-interactive-slider.js

```diff
@@ -33,13 +35,18 @@ const buildSliderTemplate = (element) => {
     <div class="i-amphtml-story-interactive-slider-container">
       <div class="i-amphtml-story-interactive-prompt-container"></div>
       <div class="i-amphtml-story-interactive-slider-input-container">
-        <input
-          class="i-amphtml-story-interactive-slider-input"
-          type="range"
-          min="0"
-          max="100"
-          value="25"
-        />
+        <div class="i-amphtml-story-interactive-side-values">0</div>
+        <div class="i-amphtml-story-interactive-slider-input-size">
+          <input
+            class="i-amphtml-story-interactive-slider-input"
+            type="range"
+            min="0"
+            max="100"
+            value="0"
+          />
+          <div class="i-amphtml-story-interactive-slider-bubble">50</div>
+        </div>
+        <div class="i-amphtml-story-interactive-side-values">100</div>
```

Comment:
Same comment as above, this is an optional UI type and needs to be removed for this PR.